doc/input-output-reference/src/overview/group-zone-forced-air-units.tex

```diff
@@ -35,6 +35,10 @@ \subsubsection{Inputs}\label{inputs-056}
 
 The name of the zone exhaust air node of the ideal loads object. This should be the same as one of the zone air exhaust nodes for the zone this component is serving. This node name is required if ZoneHVAC:IdealLoadsAirSystem is used in a zone which also has other forced air HVAC equipment. Otherwise this node name is optional but recommended.
 
+\paragraph{Field: System Inlet Air Node Name}\label{system-inlet-air-node-name}
+
+The name of the plenum outlet (or induced) air node connected to the ideal loads object. When an AirloopHVAC:ReturnPlenum is connected to the Ideal Loads Air System the Zone Supply Air Node Name should be the same as one of the zone air inlet nodes. The Zone Exhaust Air Node Name should be the same as one of the zone air exhaust nodes and one of the AirloopHVAC:ReturnPlenum inlet nodes. In the AirloopHVAC:ReturnPlenum object, for the first ideal loads air system, the System Inlet Air Node Name is entered as the plenum Outlet Node Name (required field) while all remaining ideal loads air systems, if used, list the System Inlet Air Node Name in the Induced Air Outlet Node or NodeList Name field (see Engineering Reference for schematic).
+
```

Comment:
Confused - does this connect to the plenum outlet node or the plenum induced air node?  If the plenum is shared, then what?